arbeitszeit_web/www/presenters/reject_plan_presenter.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.reject_plan import RejectPlanUseCase as UseCase
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+
+
+@dataclass
+class RejectPlanPresenter:
+    @dataclass
+    class ViewModel:
+        redirect_url: str
+
+    url_index: UrlIndex
+    notifier: Notifier
+    translator: Translator
+
+    def reject_plan(self, response: UseCase.Response) -> ViewModel:
+        if response.is_rejected:
+            self.notifier.display_info(
+                self.translator.gettext("Plan was rejected successfully")
+            )
+        else:
+            self.notifier.display_warning(
+                self.translator.gettext("Plan rejection failed")
+            )
+        return self.ViewModel(
```

Comment:
again this comes from [approve_plan_presenter.py.](https://github.com/ida-arbeitszeit/arbeitszeitapp/blob/f771d116b3ce9b7d0a419f5338eab6b686fc38f1/arbeitszeit_web/www/presenters/approve_plan_presenter.py#L31) iiuc from code perspective it is a theoretical case when an already rejected plan is rejected again. But this is currently not possible from UI view. If there are db/connection issues, I would think it is ok to redirect since user might want to try again.